### DISCUSSION TITLE
ADD RESOURCES!

### DISCUSSION THREAD

**Discussion by ReadySetGooo** [ReadySetGooo/NoNameYet#12] [Ideas]
Make a comment below of information you'd like to be added! It can be links or even just text.

**Comment by Ohrldu** [comment]
This is a list of subreddits. Idk if all are relevant. 
Aboriginal, 
Abolish slavery,
Aboriginal,
Aclu,
Against mensrights
Agriculture, 
Agriscience, 
ainbow 
Almost homeless, 
AMA, 
Anarchism, 
Anarchy101, 
Anthropology, 
Anticonsumption, 
Antifascists 
Antiwork 
Arboriculture, 
AskPolitics, 
AskAnthropology, __
Ask economics, 
Ask historians, 
Ask men, 
Ask reddit,
Asksocial science, 
Ask statistics, 
Ask women over 30
Ask men over 30, 
Assistance 
Audioboos 
Auntie Networ canada, 
Autism, 
Autism in women, 
Biostatistics, 
Black lives matter
Book exchange 
Books 
Book suggestions  
Borrow 
Budget food 
Bushcraft, UK, Germany, 
C_S_T 67 
Care 
Car living 
Change my view. 
Charity 
Chronic illness 
Church of covid, 
Citizens Climate AU 
Citizens Climate Lobby , 
Climate, 
Climate discussion,
Climate Action Plan 
Climate change,
Climate discussion, 
Climate offensive,
Climate Reality Project inactive Reddit?
Communalism 
Communism 
Communism 101. 
Composting__
Conservation __ 
Consumerism, 
Cool guides   
Cooperatives 
CoopU 
Corruption 
Couchsurfing 
Cultural anthro 
Cultural studies,
Culture list 
DAE
Data is beautiful, 
Data police 
Data science 
Debate Anarchism 
Debate Communism 
Debt 
Debtfree  
Debt strike 
Deep ecology __ 
Deep thoughts.
DemocraticSocialism 
 
Demography, 
Demsocialists  
Detrashed 
Disability 
Disability activism 
Disability rights 
Disability support 
Disabled 
Distributionism 
Divestment tiny   
DAE
Dumpster diving  
Earthstrie
Eat cheap and healthy 
Ebooks 
Effective altruism. 
Endless War 
Entrepreneur- 
Enviroaction
Environment 
Environmental Policy 
Environmental Science 
Environmentalism
Equality
Ethnobotany 
Extinction rebellion 
Farming
Fed news, 
Feminism 
Feminist theory.  
Financial aid school 
Financial independence 
Food pantry 
Food for thought (blog posts)
Food not bombs 
Foreign policy list 
Foraging 
Free food 
Frugal 
Futurology 
Gardening 
Genz 
Geopolitics 
go Fund mes
Green investor
Guerilla Gardening
Homeless

**Reply by ReadySetGooo** [reply]
Oh my gosh!! These are perfect thank you so much!!!

**Comment by Ohrldu** [comment]
This is a take action thing. Also has lessons in another part of the site. 
https://www.worldwildlife.org/pages/all-actions

Here's some systems thinking things: 
https://www.academyforchange.org/
http://www.clexchange.org/
https://stream.syscoi.com/

**Comment by ReadySetGooo** [comment]
The following are free PDFs
https://trueprepper.com/survival-pdfs-downloads/
https://cityprepping.com/blog/skill/preppers-free-pdf-library/
